### PR TITLE
Add SELinux instructions

### PR DESCRIPTION
**Pull request by scollazo**
fixes https://github.com/artefactual-labs/am-packbuild/issues/97

### CODE REVIEW

**Comment by jhsimpson** [comment]
It is possible to configure SELinux on an ubuntu machine as well, correct?  Shouldn't the SELinux configuration instructions be in a separate section of the docs, and linked to from both ubuntu and centos install sections?

**Comment by scollazo** [comment]
Selinux support for Ubuntu seems pretty outdated:

https://help.ubuntu.com/community/SELinux

Ubuntu uses AppArmor, and I'm not aware of any issues with Archivematica, so nothing to document on that side.

File: admin-manual/installation/installation.rst [hunk 1]
```diff
@@ -317,6 +317,20 @@ Update your system
 .. code:: bash
 
    sudo yum update
+   
+If your environment uses SELinux, you'll need to tweak some configurations:
```
Comment (jhsimpson) [inline admin-manual/installation/installation.rst]: Suggest adding a sentence after this

`At a minmum you will need to run the following commands.  Additional configuration may be required for your local setup.  `

**Review by jhsimpson — APPROVED** [review]
Other than a minor suggested addition, this looks good to me
:+1:

**Pull request merged**